automatic/vscode-insiders.install/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$softwareName = 'Microsoft Visual Studio Code Insiders'
+$version = '1.46.0'
+if ($version -eq (Get-UninstallRegistryKey "$softwareName").DisplayVersion) {
```

Comment:
This may fail in some cases (especially in PowerShell v2).
Please check the return value before checking `.DisplayVersion`, the return value may also be an array when multiple versions are installed (not sure if that is possible), in those cases it will also fail on PowerShell V2 as well as giving the incorrect version in rare cases.

Reply:
I removed this code, it makes no sense. It prevents `choco install vscode-insiders.install -f` to in fact reinstall an already installed.